src/main/java/org/folio/isbn/IsbnUtil.java

```diff
@@ -39,53 +39,42 @@ public static boolean isValidThirteenDigitNumber(String number) {
   /**
    * Convert an ISBN-10 code to an ISBN-13 code.
    * <p>
-   * This method requires a valid ISBN-10 with NO formatting
+   * This method accepts ISBN-10 with or without formatting
    * characters.
    *
    * @param isbn10 The ISBN-10 code to convert
    * @return A converted ISBN-13 code or <code>null</code>
    * if the ISBN-10 code is not valid
    */
   public static String convertToThirteenDigitNumber(String isbn10) {
```

Comment:
I tend to rename this method to shorter form: convertTo13DigitNumber

Reply:
done